src/dom-to-image-more.js

```diff
@@ -283,6 +287,12 @@
                 function copyStyle(source, target) {
                     if (source.cssText) {
                         target.cssText = source.cssText;
+
+                        // Fix strange box-shadow in Safari
+                        if (util.isSafari()) {
+                            target.cssText = target.cssText.replace(/box-shadow(.*?);/, 'box-shadow: none!important;');
```

Comment:
This regex seems a bit dangerous, might eat a lot more than just the box-shadow setting.